### PR TITLE
Run commands to stabilize benchmark environment

### PR DESCRIPTION
**Pull request by mariecwhite**

### CODE REVIEW

File: build_tools/benchmarks/run_benchmarks_on_android.py [hunk 1]
```diff
@@ -412,6 +426,16 @@ def main(args):
         verbose=args.verbose,
     )
 
+    # Turn off thermal throttling. CPU frequencies are throttled when the device is warm.
+    adb_execute_as_root(["setprop", "persist.vendor.disable.thermal.control", "1"])
+
+    # Fake screen-on status. Performance may be affected when screen is off.
+    # This only works on later versions of Android. Disable for now.
+    # adb_execute_as_root(["sendhint", "-m", "DISPLAY_INACTIVE", "-e", "0"])
```
Comment (antiagainst) [inline build_tools/benchmarks/run_benchmarks_on_android.py:434]: Can we check the current android version and enable if so?

File: build_tools/benchmarks/run_benchmarks_on_android.py [hunk 2]
```diff
@@ -412,6 +426,16 @@ def main(args):
         verbose=args.verbose,
     )
 
+    # Turn off thermal throttling. CPU frequencies are throttled when the device is warm.
+    adb_execute_as_root(["setprop", "persist.vendor.disable.thermal.control", "1"])
```
Comment (antiagainst) [inline build_tools/benchmarks/run_benchmarks_on_android.py:430]: It has "vendor" in its name. Is this applicable everywhere?

File: build_tools/benchmarks/run_benchmarks_on_android.py [hunk 3]
```diff
@@ -412,6 +426,16 @@ def main(args):
...
+
+    # Fake screen-on status. Performance may be affected when screen is off.
+    # This only works on later versions of Android. Disable for now.
+    # adb_execute_as_root(["sendhint", "-m", "DISPLAY_INACTIVE", "-e", "0"])
+
+    if is_pixel_device():
```
Comment (antiagainst) [inline build_tools/benchmarks/run_benchmarks_on_android.py:436]: This enables it unconditionally. It should be controlled by `args.pin_cpu_freq`?

**Review by antiagainst — COMMENTED** [review]
Nice. A few questions.

**Comment by benvanik** [comment]
Closing stale PR.

**Pull request closed without merging**